scripts/functions.sh

```diff
@@ -187,6 +188,7 @@ function common_usage {
   echo "          --icmp_code:              filter ICMP code                           (default: n/a)"
   echo "          --peer_ip:                filter peer IP                             (default: n/a)"
   echo "          --drops:                  filter flows with only dropped packets     (default: false)"
+  echo "          --regexes:                filter flows using regex                   (default: n/a)"
```

Comment:
I feel like this deserves more explanation and perhaps a simple example, no? What the regex will be running against? The json representation of the flow? From code it looks like user has to provide keys and values, we can mention that.

Also, nit: in the description I'd use the full wording "regular expressions" rather than "regex".

Reply:
Every filter need examples so I created a followup: https://issues.redhat.com/browse/NETOBSERV-2011